### ISSUE TITLE
Instead of just missing chapters show filtered and missing chapters.

### ISSUE THREAD

**Issue by THElegend5** [tachiyomiorg/tachiyomi#9259]
### Describe your suggested feature

Currently the missing chapters are detected from the filtered chapters which makes it so that if you have a filter like unread it shows all the read chapters as missing which is quite misleading as they are just filtered and not missing.
![Screenshot_2023-03-26-11-38-19-503_eu kanade tachiyomi debug](https://user-images.githubusercontent.com/63903377/227764262-453c7d7a-348e-4a1f-8ea8-af29768bbfd6.png)
A better option would be to show the filtered chaptered as filtered xx chapters and missing chapters are then detected from the current list between the lower and upper list of current available chapters.
So suppose there are a total of 60 chapters and chapters till 30 are filtered and chapter 1 , 2 ,  58 and 59 are missing then it will show Filtered 28 chapters as 1 and 2 are missing from first 30 and detect missing from the chapters which are left which are 31 to 60 and 58 and 59 being missing so Missing 2 chapters.
So in the end it would be Filtered 28 chapter and Missing 2 chapters.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by jferdous70** [comment]
Missing chapters is technically misleading for filtered chapters, since they are hidden deliberately by user from the app. In general, missing is understood as unavailable chapters on the source website. Thus, the change would be much appreciated

**Comment by molitar** [comment]
I agree it always worked that way before the update.  Now update confusing as you have to go remove filter to see if there is any actual missing chapters or not.

**Comment by WeepingClown13** [comment]
@arkon hi, is there a status update on this? Now that missing chapters count is displayed directly above the chapters (neat and lovely UI, thanks for that!), it has become especially misleading, confusing and annoying. At the very least, won't simply removing filtered chapters from missing chapters work as a hotfix?
![Screenshot_20240107_122856_Tachiyomi.png](https://github.com/tachiyomiorg/tachiyomi/assets/95921427/57b5353a-e185-4070-a6e3-67b62ad9de6d)

**Comment by arkon** [comment]
PRs are welcome.
